src/Monomer/Main/Types.hs

```diff
@@ -382,12 +383,27 @@ appDisableAutoScale disable = def {
 }
 
 {-|
-Available fonts to the application. An empty list will make it impossible to
-render text.
+Available fonts to the application, loaded from the specified path. 
+Specifying no fonts will make it impossible to render text.
 -}
-appFontDef :: Text -> Text -> AppConfig e
-appFontDef name path = def {
-  _apcFonts = [ FontDef name path ]
+appFontDefFile :: Text -> Text -> AppConfig e
+appFontDefFile name path = def {
+  _apcFonts = [ FontDefFile name path ]
+}
+
+{- |
+Alias for 'appFontDefFile' for backwards compatibility.
+-}
+appFontDef = appFontDefFile
+{-# DEPRECATED appFontDef "Use appFontDefFile directly" #-}
+
+{-|
+Available fonts to the application, loaded from the bytes in memory. 
+Specifying no fonts will make it impossible to render text.
```

Comment:
Maybe adding a short example about how to embed a font would be a good idea (the text I added is just a sample, change it as you see fit).

```haskell
{-|
Available fonts to the application, loaded from the bytes in memory. 
Specifying no fonts will make it impossible to render text.

One use case for this function is to embed fonts in the application, without the need to distribute the font files. The [file-embed](https://hackage.haskell.org/package/file-embed-0.0.15.0/docs/Data-FileEmbed.html) library can be used for this.
@
appFontDefMemory "memoryFont" $(embedFile "dirName/fileName")
@
-}
appFontDefMemory :: Text -> ByteString -> AppConfig e
```